joomla/joomla-cms
# [4.0] Namespace com_installer

Pull Request for Issue # .

### Summary of Changes
As title says, this PR converts com_installer to namespace MVC for Joomla 4.

### Testing Instructions
Install latest Joomla 4.0-dev branch, apply patch, test make sure it works in the same way as none-namespace version.
